touchnet/models/qwen2_audio/__init__.py

```
@@ -0,0 +1,70 @@
+        torch.nn.init.ones_(layer.input_layernorm.weight)
+        torch.nn.init.ones_(layer.post_attention_layernorm.weight)
+
+    # NOTE(xcsong): Do some NaN check
+    for name, param in model.named_parameters():
+        if torch.isnan(param).any() or torch.isinf(param).any():
+            raise ValueError(f"NaN/inf in model parameters `{name}`.")
+
+
+def get_num_flop_per_token(num_params: int, model_config: Qwen2AudioConfig, seq_len: int) -> int:
+    # NOTE(xcsong): We do not include flops from speech_encoder
+    l, h, q, t = (
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)